common/src/main/java/no/difi/meldingsutveksling/config/KeyStoreProperties.java

```diff
@@ -0,0 +1,40 @@
+package no.difi.meldingsutveksling.config;
+
+import lombok.Data;
+import lombok.ToString;
+import org.springframework.core.io.Resource;
+
+import javax.validation.constraints.NotNull;
+import java.security.KeyStore;
+
+@Data
+@ToString(exclude = "password")
+public class KeyStoreProperties {
+
+    /**
+     * Type of KeyStore
+     *
+     * Examples: JKS, Windows-MY
+     */
+    private String type = KeyStore.getDefaultType();
+
+    /**
+     * Keystore alias for key.
+     */
+    @NotNull
+    private String alias;
+
+    /**
+     * Path of jks file.
+     *
+     * May be empty if type = Windows-MY
+     */
+    private Resource path;
+
+    /**
+     * Password of keystore and entry.
+     */
+    @NotNull
+    private String password = "";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2068)